regression-test/suites/load_p0/http_stream/test_http_stream.groovy

```diff
@@ -600,231 +600,43 @@ suite("test_http_stream", "p0") {
         try_sql "DROP TABLE IF EXISTS ${tableName13}"
     }
 
-    // 14. test parquet orc case
-    def tableName14 = "test_parquet_orc_case"
+    // 14. test label 
+    def tableName14 = "test_http_stream_label"
+    def label = label = UUID.randomUUID().toString().replaceAll("-", "")
```

Comment:
def label = UUID.randomUUID().toString().replaceAll("-", "")?

Reply:
This label is used in SQL and will be compared with the returned results

Reply:
Why label is assigned twice:  
`def  label = label = `